developer_manual/core/theming.rst

```diff
@@ -222,6 +222,11 @@ Trash can image                                                  ``owncloud/core
 .. note:: 
    When overriding the favicon, make sure your custom theme includes and override for both ``owncloud/apps/core/img/favicon.svg`` and ``owncloud/apps/core/img/favicon.png``, to cover any future updates to favicon handling.
 
+.. note::   
+    When using different icons for file types in a custom theme it is necessary to use ``occ maintenance:mimetype:update-js`` for them to get used. For more information please refer to `mimetypes management`_.
+    
```

Comment:
Proposal:
When using own icons for file types than the shipped once in a custom theme, it is necessary to run ``occ maintenance:mimetype:update-js`` to activate them. For ...

Reply:
I'd suggest:

> When using custom filetype icons in a custom theme, it is necessary to run ``occ maintenance:mimetype:update-js`` to activate them. For more information please refer to `mimetypes management`_.

Reply:
@settermjd 👍